website/docs/docs/core/connect-data-platform/mssql-setup.md

```diff
@@ -4,7 +4,7 @@ description: "Read this guide to learn about the Microsoft SQL Server warehouse
 id: "mssql-setup"
 meta:
   maintained_by: Community
-  authors: 'dbt-msft community (https://github.com/dbt-msft)'
+  authors: 'Mikael Ene'
```

Comment:
```suggestion
  authors: 'Mikael Ene & [dbt-msft community](https://github.com/dbt-msft)'
```

Reply:
Hi @dataders . Long time yes :-). I think _maintained_by:_ explains who are maintaining the project. If the dbt-msft community are maintaining this, it should be stated there I guess? I mean, I did created this adapter and was the only maintainer for the first year. Previously my name was stated as author but was just removed one day, without any notification. It is a little weird to change author according to who currently maintains the project don't you think?